content/ja/handbook/engineering/readmes/kamil-trzcinski.md

```
@@ -0,0 +1,112 @@
+
+私が行うすべてのこと（自分でやるときも誰かの変更をレビューするときも）において、常に与えられた変更の周辺のコードをわずかに改善することを目指しています。私の考えは、技術的な負債の解消に 10〜20% 多くの時間をかけることで、リファクタリングを特別に行わずに常にコードベースをリファクタリングできるということです。リファクタリングのスケジューリングについての私の認識は、通常は悪いアイデアだということです。新機能の作業中にコードベースを常にリファクタリングする方がはるかに簡単だと感じています。私と一緒に働く誰にでも期待しているのは: 将来の生活を楽にするために少し余分にやることです。
+
+上記のアプローチのいくつかの例:
+
+* 発見したが明確でなかったものにコメントをつける
+* 既存の実装のアーキテクチャを改善する
+* メソッドとテストを書き直して読みやすくするか、より高性能にする
+* 発見した問題について Issue を作成する
+* この変更の前に一部の側面を改善するための小さなマージリクエストを行う
+
+## 私のフォーカス
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)